java/Ice/greeter/client/src/main/java/com/zeroc/greeter/Client.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.greeter;
+
+import com.zeroc.Ice.Communicator;
+import com.zeroc.Ice.Util;
+import com.zeroc.greeter.VisitorCenter.GreeterPrx;
+import java.util.concurrent.CompletableFuture;
+
+class Client {
+    public static void main(String[] args)
+    {
+        // Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+        try(com.zeroc.Ice.Communicator communicator = com.zeroc.Ice.Util.initialize(args))
```

Comment:
Indeed, there should be a space here.

Looking at other demos though, it seems like the demos use our old Java style,
before we switched to all these other spotless styles.

There's also no need to qualify `Communicator` since it's imported.
I personally would leave the qualification on `Util`, since `util` is very common, even in the Java Standard Library.
```suggestion
        try (Communicator communicator = com.zeroc.Ice.Util.initialize(args))
```